addons/account_asset/report/account_asset_report.py

```diff
@@ -26,9 +26,9 @@ class AssetAssetReport(models.Model):
     unposted_value = fields.Float(string='Unposted Amount', readonly=True)
     company_id = fields.Many2one('res.company', string='Company', readonly=True)
 
-    def init(self, cr):
-        tools.drop_view_if_exists(cr, 'asset_asset_report') # why is this called if the instruction 2 lines below is create or REPLACE view ?
-        cr.execute("""
+    @api.model
```

Comment:
This seems incorrect, old definition was an api.model_cr. There was no context, was it missing somewhere ?